rpcs3/Emu/Io/GHLtar.cpp

```diff
@@ -0,0 +1,204 @@
+﻿#include "stdafx.h"
+#include "GHLtar.h"
+#include "Emu/Cell/lv2/sys_usbd.h"
+#include "Input/pad_thread.h"
+#include <Emu\Cell\Modules\cellPad.cpp>
+#include <Emu\RSX\GL\GLHelpers.h>
```

Comment:
Don't mix the quotes and the chevrons. Don't mix slashes either. This is why it won't pass CI I'm p sure

Reply:
ah that makes sense - I just chose the defaults that Visual Studio offered me so I'll go and fix that right now